internal/pkg/cli/flag.go

```diff
@@ -75,6 +75,7 @@ const (
 	commandFlag        = "command"
 	entrypointFlag     = "entrypoint"
 	taskDefaultFlag    = "default"
+	generateCMDFlag    = "generate-cmd"
```

Comment:
Nit: hmm, this appears like an initialism (https://github.com/golang/go/wiki/CodeReviewComments#initialisms) but it isn't. Maybe `generateCommandFlag`?